tests/python/gaia-ui-tests/gaiatest/apps/system/regions/fxaccounts.py

```diff
@@ -68,6 +70,7 @@ def wait_for_successful_login(self):
         return done
 
     def tap_done(self):
+        self._switch_to_fxa_iframe()
         done = self.wait_for_successful_login()
         done.tap()
         #Switch back to the settings app
```

Comment:
Correct, I was thinking about creating a separate bug for fxa page related changes, but I thought changes were small enough to be added to this PR.  (And without this changes, I wouldn't know whether the return_to_prev_menu() method is working properly in fxa page, since it works within its own iframe)  Will try to separate the works in the future.